docs/docs/Starter-Projects/starter-projects-vector-store-rag.md

```diff
@@ -60,10 +60,20 @@ The **Retriever Flow** (top of the screen) embeds the user's queries into vecto
 	1. In the **Astra DB Application Token** field, add your **Astra DB** application token.
 	The component connects to your database and populates the menus with existing databases and collections.
 	2. Select your **Database**.
+	If you don't have a collection, select **New database**.
+	Complete the **Name**, **Cloud provider**, and **Region** fields, and then click **Create**. **Database creation takes a few minutes**.
 	3. Select your **Collection**. Collections are created in your [Astra DB deployment](https://astra.datastax.com) for storing vector data.
-	If you don't have a collection, see the [DataStax Astra DB Serverless documentation](https://docs.datastax.com/en/astra-db-serverless/databases/manage-collections.html#create-collection).
-	4. Select **Embedding Model** to bring your own embeddings model, which is the connected **OpenAI Embeddings** component.
-	The **Dimensions** value must match the dimensions of your collection. You can find this value in the **Collection** in your [Astra DB deployment](https://astra.datastax.com).
+	:::info
+	If you select a collection embedded with Nvidia through Astra's vectorize service, the **Embedding Model** port is removed, because you have already generated embeddings for this collection with the Nvidia `NV-Embed-QA` model. The component fetches the data from the collection, and uses the same embeddings for queries.
+	:::
+	If you don't have a collection, select **New collection**.
+	Complete the **Name**, **Embedding generation method**, **Embedding model**, and **Dimensions** fields, and then click **Create**.
+	* To use embeddings generated by a provider through Astra's vectorize service, select the model from the **Embedding generation method** dropdown menu, and then select the model from the **Embedding model** dropdown menu.
+	* To use embeddings generated by a component in Langflow, select **Bring your own** for both the **Embedding generation method** and **Embedding model** fields. In this starter project, the embeddings method and model is the **OpenAI Embeddings** component connected to the **Astra DB** component.
+	* The **Dimensions** value must match the dimensions of your collection. You can find this value in the **Collection** in your [Astra DB deployment](https://astra.datastax.com).
```

Comment:
This is a lot of information for one step. It's overwhelming.

Reply:
Split and added some context